src/Symfony/Component/Intl/Tests/NumberFormatter/AbstractNumberFormatterTest.php

```diff
@@ -384,7 +384,6 @@ public function formatFractionDigitsProvider()
             [1.123, '1.1', 1, 1],
             [1.123, '1.12', 2, 2],
             [1.123, '1.123', -1, 0],
-            [1.123, '1', 'abc', 0],
```

Comment:
`'abc'` is passed as the `value` of the `setAttribute` method here:

https://github.com/symfony/symfony/pull/32525/files#diff-2a5ae15ec68c5d666ce08a09d75690efR564

It's type-hinted at `int` so it throws a TypeError, however method's logic seems to handle case where it could be a string:

https://github.com/symfony/symfony/blob/2c9a1960a164a857cd551881b580266bc77bdafa/src/Symfony/Component/Intl/NumberFormatter/NumberFormatter.php#L867

I decided to keep the type hint as `int` and remove the use case of a `string` value